projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/prompt/index.ts

```diff
@@ -251,6 +252,9 @@ Strictly follow those rules:
 - When the user provides instructions, translate them into appropriate Gutenberg blocks and Jetpack form structure.
 - Avoid sensitive or controversial topics and ensure your responses are grammatically correct and coherent.
 - If you cannot generate a meaningful response to a user's request, reply only with "__JETPACK_AI_ERROR__". This term should only be used in this context, it is used to generate user facing errors.`,
+			context: {
+				type: 'form-ai-extension-system-prompt',
```

Comment:
We should use a TS type for this and another context type. Please don't take it as a blocker. Let's do it in a follow-up.

Reply:
Sure! The next step involves moving more complex prompts, so I am planning to type everything since the context property will be around a lot of places. Thank you for the suggestion!